feat: remove karma and jasmine

asen-arizanov opened the pull request.

mitagg commented:
The following message won't produce a release, in order to produce a major version bump, the body of the commit should contains "Breaking change" message. You could use the [following commit](https://github.com/telerik/kendo-react-tasks/commit/8e158545) as reference. I would also change the "type" from chore to feat.

asen-arizanov committed:
feat: Remove Karma and Jasmine

Remove Karma and Jasmine in favour of Jest

BREAKING CHANGE: Remove gulp test tasks
